docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -12,6 +12,19 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/react-sdk                   | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2023-03-14
+
+Released: `@100mslive/hms-video-store@0.9.0`, `@100mslive/react-sdk@0.7.0`
+
+#### Added:
+- `getAuthTokenByRoomCode` in hmsActions to get token for room short code.
```

Comment:
can we link to api reference if it has detailed tsdoc?
also we should have a section for this in generating auth token page.